Makefile

```diff
@@ -16,22 +16,25 @@ all: build test verify
 
 # Define some constants
 #######################
-ROOT          = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-BINDIR       ?= bin
-COVERAGE     ?= $(CURDIR)/coverage.html
-SC_PKG        = github.com/kubernetes-incubator/service-catalog
-TOP_SRC_DIRS  = cmd contrib pkg util
-SRC_DIRS      = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*.go \
-                  -exec dirname {} \\; | sort | uniq")
-TEST_DIRS     = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
-                  -exec dirname {} \\; | sort | uniq")
-GO_VERSION    = 1.7.3
-GO_BUILD      = go build -i -v
-BASE_PATH     = $(ROOT:/src/github.com/kubernetes-incubator/service-catalog/=)
-export GOPATH = $(BASE_PATH):$(ROOT)/vendor
-DOCKER_CMD    = docker run --rm -ti -v $(PWD):/go/src/$(SC_PKG) \
-                 -e GOOS=$$SC_GOOS -e GOARCH=$$SC_GOARCH \
-                 scbuildimage
+ROOT           = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+BINDIR        ?= bin
+COVERAGE      ?= $(CURDIR)/coverage.html
+SC_PKG         = github.com/kubernetes-incubator/service-catalog
+TOP_SRC_DIRS   = cmd contrib pkg util
+SRC_DIRS       = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*.go \
+                   -exec dirname {} \\; | sort | uniq")
+TEST_DIRS      = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
+                   -exec dirname {} \\; | sort | uniq")
+NEWEST_GO_FILE = $(shell find $(SRC_DIRS) -name \*.go -printf '%T+ %p\n' \
+                   | sort -r | head -n 1 | sed "s/.* //")
+TYPES_FILES    = $(shell find pkg/apis -name types.go)
```

Comment:
The only thing that changes in this block is the addition of "NEWEST_GO_FILE" and "TYPES_FILES". The rest are here because I twiddled the spacing.